src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj

```diff
@@ -86,12 +98,24 @@
     <EmbeddedResource Include="TestFiles\R4\AuthConfigWithValidRoles.json" />
     <EmbeddedResource Include="TestFiles\R4\BasicExampleNarrative.json" />
     <EmbeddedResource Include="TestFiles\R4\BasicExampleNarrative.xml" />
+    <EmbeddedResource Include="TestFiles\R4\Condition-For-Patient-f001.json" />
```

Comment:
Should these be moved out of common and put into R4 and Stu3 projects respectively?

Reply:
We could possibly do this in another pass. At least in `Tests.Common` they are only compiled/embedded once. This assembly is shared with all test projects